estuary/api/v1.py

```diff
@@ -337,6 +339,7 @@ def get_artifact_relationships(resource, uid, relationship):
         serialized_node = node.serialized_all
         serialized_node['resource_type'] = node.__label__
         serialized_node['display_name'] = node.display_name
+        serialized_node['timeline_property'] = node.timeline_property
```

Comment:
Depending on the `relationship` that is requested, this will raise a `NotImplementedError`. One example would be requesting `/api/v1/relationships/freshmakerevent/1234/requested_builds`. This line could probably just be deleted since it's not important for this API endpoint.